### PR TITLE
Fix #21 feature request: Add bullet gem with generate command

### LINKED ISSUE #21: [Feature Request]: Add bullet gem
### Is your feature request related to a problem? Please describe.

This gem makes sure that we write optimised queries

### Describe the solution you'd like

Reference: https://youtu.be/qyoRTNKsEGU

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by rray524**
Resolves #21 

### Description
Feature request about adding bullet gem to the project

### Type of change
- [ ] Feature

### How to test this PR?
Do bundle install, and enable the Bullet gem with generate command

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -1,6 +1,15 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable        = true
+    Bullet.alert         = true
```
Comment (rt4914) [inline config/environments/development.rb:6]: Keep this false.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Just one minor change.

File: config/environments/development.rb (hunk 1 above)
Comment (rray524) [reply config/environments/development.rb]: hi @rt4914 i kept both false now. thanks

File: Gemfile [hunk 2]
```diff
@@ -48,6 +48,8 @@ group :development do
 
   # A gem that calculates the exchange rate using published rates from open-exchange-rates
   gem 'money-open-exchange-rates'
+  #The Bullet gem is designed to help you increase your application's performance by reducing the number of queries it makes. It will watch your queries while you develop your application and notify you when you should add eager loading (N+1 queries), when you're using eager loading that isn't necessary and when you should use counter cache.
```
Comment (rt4914) [inline Gemfile:51]: Remove this comment, we actually don't need this.

File: config/environments/development.rb [hunk 3]
```diff
@@ -1,6 +1,15 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable        = false
```
Comment (rt4914) [inline config/environments/development.rb:5]: This should be enabled otherwise it won't work.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@rray524 Suggested changes.

File: Gemfile (hunk 2 above)
Comment (rray524) [reply Gemfile]: OK removed. thanks

File: config/environments/development.rb (hunk 3 above)
Comment (rray524) [reply config/environments/development.rb]: sure, enabled it to true.

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**